### PR TITLE
Switch to use MUCs for videobridge

### PR DESCRIPTION
**Pull request by doobry-systemli**
Upstream already switched to MUCs for videobridging some releases ago, this PR makes our role follow.

### CODE REVIEW

**Comment by ramondelafuente** [comment]
Looks good to me, I saw the same difference in configuration earlier today.

Seeing the failed build; is that a fluke? Or maybe it makes sense to merge #5 and rebase?

File: tasks/nginx.yml [hunk 1]
```diff
@@ -3,12 +3,22 @@
...
-    src: nginx.conf.j2
+    src: nginx/sites-available/vhost.conf.j2
     owner: root
     group: root
     mode: "0644"
   notify: restart nginx
 
+- name: Copy Nginx multiplexer config if local turnserver is used.
+  template:
+    dest: "/etc/nginx/modules-enabled/60-jitsi-meet.conf"
```
Comment (ramondelafuente) [inline tasks/nginx.yml:14]: Should this be a write to `modules-available` and then a symlink?

**Comment by 0x46616c6b** [comment]
> > I re-run the tests (https://travis-ci.com/github/systemli/ansible-role-jitsi-meet/jobs/321684316) and saw the following error:
> > ```
> > fatal: [instance]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'jitsi_meet_videobridge_muc_nickname' is undefined"}
> > ```
> 
> Hmm.. I could not figure out what value needs to be there, as in my configuration it's a uuid.

The variable `jitsi_meet_videobridge_muc_nickname` is missing in defaults @doobry-systemli 

I guess we can fix this quickly.

File: tasks/nginx.yml (hunk 1 above)
Comment (doobry-systemli) [reply tasks/nginx.yml:14]: Thanks for the review, @ramondelafuente! I decided to implement it exactly the way the Debian package does it in order to keep the delta as small as possible. That's why I didn't do the symlink dance here ;)

**Comment by doobry-systemli** [comment]
> I re-run the tests (https://travis-ci.com/github/systemli/ansible-role-jitsi-meet/jobs/321684316) and saw the following error:
> 
> ```
> fatal: [instance]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'jitsi_meet_videobridge_muc_nickname' is undefined"}
> ```

Oh sorry. Added it now.

**Pull request merged**